glean_parser/schemas/pings.1-0-0.schema.yaml

```diff
@@ -0,0 +1,53 @@
+$schema: http://json-schema.org/draft-07/schema#
+title: Pings
+description: |
+  Schema for the pings.yaml files for Mozilla's glean telemetry SDK.
+
+  The top-level of the `pings.yaml` file has a key defining the name of each
+  ping. The values contain metadata about that ping. Ping names must be
```

Comment:
This seems to contrast with the `tests/data/pings.yaml`, which is currently using *camelCase*. Is the schema validation broken?

Reply:
The test was broken -- but validation is actually working.  Should be fixed (and enforced correctly) now.